modules/tide_edit_protection/config/optional/views.view.tide_locked_content.yml

```diff
@@ -9,6 +9,8 @@ dependencies:
     - user.role.site_admin
   module:
     - content_lock
+    - content_lock_timeout
+    - tide_edit_protection
     - node
     - user
 id: tide_locked_content
```

Comment:
@vincent-gao curiosity, any reason why moving this from install directory to optional directory. Do the settings get imported to config/sync from an optional directory on its first install? I mean if some other project installing this module for the first time, will this view get installed from this optional directory?

Reply:
@MdNadimHossain because it was incorrect to put the view into `install` folder. both install/optional are for installing configurations. `install` directory is for configurations that doesn't need (or doesn't care ) dependencies, while the configurations in `optional` is the opposite.

Reply:
by adding the changes, the installing order will be

content_lock -> content_lock_timeout -> tide_edit_protection -> disable views.view.locked_content view -> install views.view.tide_locked_content view